docs/tutorial/ipc.md

```diff
@@ -439,6 +439,8 @@ After loading the preload script, your renderer process should have access to th
 :::caution Security warning
 We don't directly expose the whole `ipcRenderer.on` API for [security reasons][]. Make sure to
 limit the renderer's access to Electron APIs as much as possible.
+Also don't just pass the callback to `ipcRenderer.on` as this will leak `ipcRenderer` via `event.sender`.
+Use a custom handler that invoke the `callback` only with the desired arguments.
```

Comment:
To Sam's offline comments, should also mention that you want to be intentional about which parameters you pass to the callback since the IPC parameters might be for internal (in the preload) usage only. Don't have a suggested wording, just making a note.